vpr/src/server/pathhelper.h

```diff
@@ -18,15 +18,34 @@ namespace server {
  * It contains the critical path list and the generated report as a string.
 */
 struct CritPathsResult {
+    /**
+    * @brief Checks if the CritPathsResult contains report.
+    * @return True if contains report, false otherwise.
+    */
     bool is_valid() const { return !report.empty(); }
+
+    /**
+    * @brief Vector containing timing paths.
+    */
     std::vector<tatum::TimingPath> paths;
+
+    /**
+    * @brief String containing the generated report.
+    */
     std::string report;
 };
 using CritPathsResultPtr = std::shared_ptr<CritPathsResult>;
 
-/** 
- * @brief Helper function to calculate critical path timing report with specified parameters.
- */
+/**
+* @brief Calculates the critical path.
+
+* This function calculates the critical path based on the provided parameters.
+* @param type The type of the critical path.
+* @param crit_path_num The max number of the critical path.
```

Comment:
Change to "The max number of critical paths to record"
I suggest also listing the valid options for the type of critical path and the level of detail for the timing report here, or adding a ref to where you defined that in the overall (index.rst) documentation.